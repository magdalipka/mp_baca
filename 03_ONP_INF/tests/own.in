20
INF: a+b*(c/d)+g^r-p/(m+n)
INF: b=y+h*c-y/g^k
INF: a~+b*c
INF: ()+b
INF: (a +,b..*c)/(b/c+a*d)
INF: a=b<c+y*t<(c+d)
INF: l^(a*c+y*(m+o))
INF: (a+b)*(c+d)/y(
INF: a^b^m*(c+d)
INF: ~(~(~((~a+~~b)/~c)*~d)^~e)
ONP: abc-*ab^c/+
ONP: a~~~~
ONP: ab+~
ONP: ab+~~
ONP: ab+c/gb*cd/*-
ONP: b~cd/~+g^h^
ONP: abcdefg+*-/%
ONP: a~b~~+c~/~d~*~e~^~
ONP: xabcd^^=
ONP: klmo**/tr%+
