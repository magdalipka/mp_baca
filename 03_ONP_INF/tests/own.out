ONP: a b c d / * + g r ^ + p m n + / -
ONP: b y h c * + y g k ^ / - =
ONP: error
ONP: error
ONP: a b c * + b c / a d * + /
ONP: a b c y t * + < c d + < =
ONP: l a c * y m o + * + ^
ONP: error
ONP: a b m ^ ^ c d + *
ONP: a ~ b ~ ~ + c ~ / ~ d ~ * ~ e ~ ^ ~
INF: a * ( b - c ) + a ^ b / c
INF: ~ ~ ~ ~ a
INF: ~ ( a + b )
INF: ~ ~ ( a + b )
INF: ( a + b ) / c - g * b * ( c / d )
INF: ( ( ~ b + ~ ( c / d ) ) ^ g ) ^ h
INF: error
INF: ~ ( ~ ( ~ ( ( ~ a + ~ ~ b ) / ~ c ) * ~ d ) ^ ~ e )
INF: error
INF: k / ( l * ( m * o ) ) + t % r
